server/main.go

```diff
@@ -11,18 +12,21 @@ import (
 )
 
 var (
-	Version   string
-	Buildtime string
+	Version   string = "dev"
```

Comment:
don't do this for version just use the gitversion

Reply:
@MaartendeKruijf, the git Version is set during compilation. When you dev using `make dev` the default value for version will be `dev`

Reply:
This is not logical you're still building a git version if the version is for release it will be a tag version. If it's a dev version you will just see 1.0.0-1-38jf83fj3j-dirty